e2e-tests/src/auth/checkout.npg-payment-auth.integration.test.ts

```diff
@@ -36,15 +37,45 @@ describe("Checkout login and payment flow", () => {
     /**
      * Step 1: Login
      */
-    it("Should successfully login in checkout", async () => {
-        await page.waitForSelector('#login-header button');
-        await page.locator('#login-header button').click();
+    const uatLogin = async () => {
```

Comment:
```suggestion
    const oneIdentityLogin = async () => {
```

just make clear in method name that we are handling OneIdentity login steps here

Reply:
Solvd in https://github.com/pagopa/pagopa-checkout-tests/pull/112/commits/b850dec38eb7bb49063f848cab46e3b3a9789f17